Automattic/wp-calypso
# Site Migration: Auto-add trial plan after email verification

Related to #89254

## Proposed Changes

* Add a new stepper step that assigns the migration trial to the site.
* After email verification succeeds, move to the add migration trial step.
* Once the trial has been added, move to the bundle transfer step.

## Testing Instructions



* Switch to this PR
* Navigate to `/start` on an account that does *not* have a verified email address.
* Select Import at the goals step.
* Put in your URL, select "Everything", and choose the free trial at the upgrade step
* You'll be brought to the email verification step.
* Verify your email by following the link in the email.
* The email verification step should automatically forward you to the processing step.
* You should land on the site migration instructions page.
* If you visit your site, you should see it has successfully migrated to Atomic with a trial plan.
* Ensure you can still complete the flow for a site with an already verified email, and with a paid plan.

## Pre-merge Checklist
